edi_punchout/data/uom_uom.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2023 Hunki Enterprises BV
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+
+    <record id="uom.product_uom_unit" model="uom.uom">
```

Comment:
This file contains the same data as the oca module `uom_unece`.
https://github.com/OCA/community-data-files/blob/13.0/uom_unece/data/unece.xml
Maybe easier to depend on that module?

Reply:
no, that's conceptually different things. Unece just happens to have the the same value here.